util/validation.go

```diff
@@ -70,3 +76,7 @@ func GetCountryCode(prefix string, phone string) (string, error) {
 
 	return countryCode, nil
 }
+
+func FilterField(field string) bool {
```

Comment:
Why old func is not removed? 
![image](https://github.com/casdoor/casdoor/assets/3787410/0ab42133-59da-46f4-8c4f-cc9e5ef198d5)